saleor/plugins/vatlayer/plugin.py

```diff
@@ -179,6 +173,35 @@ def calculate_checkout_total(
             checkout_info, lines, address, discounts
         )
 
+    def calculate_order_total(
+        self,
+        order: "Order",
+        lines: Iterable["OrderLine"],
+        previous_value: TaxedMoney,
+    ) -> TaxedMoney:
+        if self._skip_plugin(previous_value):
+            return previous_value
+
+        currency = order.currency
+
+        total = zero_taxed_money(currency)
+        undiscounted_total = zero_taxed_money(currency)
+        for line in lines:
+            total += line.total_price
+            undiscounted_total += line.undiscounted_total_price
+        total += order.shipping_price
+
+        # Vatlayer doesn't propagate order discount to shipping we should include
+        # remaining amount in total calculation.
+        order_discount = order.discounts.filter(type=OrderDiscountType.MANUAL).first()
+        if order_discount and order_discount.amount > undiscounted_total.gross:
+            import ipdb
+
+            ipdb.set_trace()
```

Comment:
🚓

Reply:
also lines 197-202 are not covered by tests, that's how the ipdb sneaked through tests unseen ;)